src/util/Constants.js

```diff
@@ -374,6 +374,7 @@ exports.InviteScopes = [
  * * APPLICATION_COMMAND
```

Comment:
Shouldn't this be renamed to SLASH_COMMAND like it was on the upstream PR? Now that context menus are a thing, having an APPLICATION_COMMAND message type can be quite vague

Reply:
That would be a breaking change, it'll have to wait for v14

Reply:
pretty sure there was a PR before v13 to change that, why wasn't that merged 🤔